packages/vue/src/components/atoms/SfSticky/SfSticky.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../css/variables';
+
+.sf-sticky {
+ position: sticky;
```

Comment:
Indentation isn't correct here.
I'm also not sure if it's a good idea to use `position: sticky`, it's not fully supported yet, don't you think is better to handle sticky position with JS?

Reply:
https://caniuse.com/#search=sticky

Reply:
Ok, you have right, I be convinced that support is greater. My mistake, I fix it

Reply:
@leomp12 I looked can i use again. Sticky doesnt work on IE and browesrs wersion before 2016. All partial support not appear to suport sticky table headers but we doesyn't .  It's to small support for us?

Reply:
@przspa imo it's important to be fully supported from IE11

Reply:
in general, modern libraries maintain support from IE11, and position sticky isn't supported...
you think we can discard IE11? wdyt @filrak ?

Reply:
IE11 is still supported by Microsoft therefore we should support it at least at the level that could be transpiled via some postcss plugins. Not sure if it's available for sticky property

Reply:
@leomp12 @filrak ok then I do that by JS :)

Reply:
@filrak @leomp12 now should be worked